app/main.py

```diff
@@ -2,24 +2,83 @@
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __set_name__(self, owner: int, value: int) -> str:
+        self.value = value
+
+    def __get__(self, instance: int, owner: str) -> None:
+        return instance.owner
+
+    def __set__(self, instance: str) -> str:
+        if self.min_amount <= self.value <= self.max_amount:
+            return True
+        else:
+            return False
 
 
 class Visitor:
-   pass
+    def __init__(
+            self, age: int, height: int, weight: int,
+            name: str = ""
+    ) -> None:
+        self.age = age
+        self.height = height
+        self.weight = weight
+        self.name = name
 
 
 class SlideLimitationValidator(ABC):
-    pass
+    def __init__(self, age: int, height: int, weight: int) -> None:
+        self.age = age
+        self.height = height
+        self.weight = weight
 
 
-class ChildrenSlideLimitationValidator(SlideLimitationValidator):
-    pass
+class ChildrenSlideLimitationValidator(SlideLimitationValidator, IntegerRange):
```

Comment:
No need to inherit from IntegerRange
Fix in all places
IntegerRange is Descriptor